sunpy/net/attr.py

```diff
@@ -177,13 +212,20 @@ def update_values(cls, adict):
         >>> attrs.Instrument.aia, attrs.Instrument.hmi # doctest: +SKIP
         (<Instrument('AIA')>, <Instrument('HMI')>)
         """
-
-        for key, value in adict.items():
-            if isiterable(value) and not isinstance(value, str):
-                for pair in value:
-                    if len(pair) != 2:
-                        raise ValueError(f'Invalid length (!=2) for values: {value}.')
-                    else:
+        for client in adict.keys():
```

Comment:
It looks like the structure of `adict` that must be handed in here has changed, but the docs haven't. (so the docs above need updating)

Reply:
Fixed.